packages/core/src/chains/cosmos/sdkCosmos/wrapper/utils.ts

```
@@ -333,3 +333,15 @@ export function getADR36SignDoc(
     memo: '',
   };
 }
+
+export function encodeSecp256k1Pubkey(pubkey: Uint8Array): ICosmosStdPublickey {
+  if (pubkey.length !== 33 || (pubkey[0] !== 0x02 && pubkey[0] !== 0x03)) {
+    throw new Error(
+      "Public key must be compressed secp256k1, i.e. 33 bytes starting with 0x02 or 0x03"
```

ESLint

[error] Replace `"Public·key·must·be·compressed·secp256k1,·i.e.·33·bytes·starting·with·0x02·or·0x03"` with `'Public·key·must·be·compressed·secp256k1,·i.e.·33·bytes·starting·with·0x02·or·0x03',`

(prettier/prettier)